dealroom_firestore_connector/__init__.py

```diff
@@ -455,3 +471,121 @@ def __log_exception(error_code, ref, identifier, was_retried=False):
         error_logger(message, error_code)
     else:
         logging.error(f"[Error code {error_code}] {message} Retrying...")
+
+
+def get_people_doc_refs(
+    db: firestore.Client,
+    field_name: str,
+    operator: str,
+    field_value,
+):
```

Comment:
I thiiiink, I prefer if you could implement here the conditions and have the caller to define only how what he is looking for.
E.g.
```python
# Pseudo-Interface
def get_people_doc_refs(db: firestore.Client, by: Enum["linkedin", "dealroom_id"], value: Any)

# Function caller
get_people_doc_refs(db, by="linkedin", value="https://linkedin.com/asds"
```

What do you think? Not really a big deal.

Reply:
Hey, I know exactly what you mean, actually this level of conditions will be like this on the service. I consider calling this a "low level" operation and I didn't want to give it any restrictions.

Reply:
okay! makes sense